lib/class-wp-rest-plugins-controller.php

```diff
@@ -57,16 +57,71 @@ public function get_items( $request ) {
 		$data = array();
 
 		require_once ABSPATH . '/wp-admin/includes/plugin.php';
-		foreach ( get_plugins() as $obj ) {
+		$plugins = get_plugins();
+
+		// Exit early if empty set.
+		if ( empty( $plugins ) ) {
+			return rest_ensure_response( $data );
```

Comment:
let's return `return array()` here, simpler to read - no need to `rest_ensure_response`